test/central_planner_test.py

```diff
@@ -23,6 +23,7 @@
 
 luigi.notifications.DEBUG = True
 WORKER = 'myworker'
+HOST = 'localhost'
```

Comment:
Sincerely... I'm not sure. The RemoteScheduler doesn't work properly without it. At first, I thought it was needed by the RemoteScheduler by design, but looking back at the code, it doesn't seem. I didn't have time to investigate the source of the issue.